### ISSUE TITLE
Formating an array of structured values fails

### ISSUE THREAD

**Issue by kaelev7** [Barteling/STweep#18]
If I run STweep on the declaration of a Function block with a pointer to an Array of structured type it will lose the type declaration for the array.
   

```    
FUNCTION_BLOCK FB_STweepTest
VAR
	pStateBuffer		:	POINTER TO ARRAY[0..EC_MAX_SLAVES] OF ST_EcSlaveState; 
END_VAR   

FUNCTION_BLOCK FB_STweepTest
VAR
    pStateBuffer		:	POINTER TO ARRAY[0..
    EC_MAX_SLAVES] OF;
END_VAR   
``` 
Is there a reason for the line break in the size declaration?   

* STweep for TwinCAT V1.0.8   
* STweep Settings file: [STweepSettings.txt](https://github.com/Barteling/STweep/files/5885733/STweepSettings.txt)   
* TwinCAT 3.1.4024.12 
* Microsoft Visual Studio Community 2019 Version 16.8.4

**Comment by Barteling** [comment]
Thanks for reporting, I'll check what is going on.

**Comment by Barteling** [comment]
Fixed in release 2.0.1.
https://www.stweep.com/Releases/
